sdk/monitor/azure-opentelemetry-exporter-azuremonitor/README.md

```diff
@@ -61,8 +61,22 @@ System.out.println(resultCode.isSuccess());
 ```
 
 ## Key concepts
-For more information on the OpenTelemetry project, please review the [OpenTelemetry Specifications
-][opentelemetry_specification].
+
+Some of the key concepts for the Azure monitor exporter include:
```

Comment:
The product name is `Azure Monitor` right? So, should this be:

```suggestion
Some of the key concepts for the Azure Monitor exporter include:
```

Reply:
Fixed.